UN-OCHA/unocha-site
# [UNO-621] Import countries from RW

Refs: UNO-621

This PR adds the `unocha_migrate` module which provides, for now, a `unocha-migrate:update-country-terms` drush command to update or create country terms based on the ReliefWeb API countries.

*Note: this cannot be done on install when the module is enabled via a config import on a fresh install because the taxonomy entity types and vocabularies are not yet created and is also a bit brittle on existing sites. So better to run the drush command manually.*

## Tests

1. Checkout the branch, run `drush cr` and `drush cim`.
2. Run `drush unocha-migrate:update-country-terms` to update/create the terms.
3. Check that the country terms were created and, edit some of them to confirm they have name, shortname, iso3 and location coordinates.
